README.md

```diff
@@ -17,13 +17,15 @@ This site was built using [GitHub Pages](https://https://ismailakinkunmi.github.
 
 ## Usage
 
-start by cloning this repository by running:
+- start by cloning this repository by running:
 
-git clone git@github.com:ismailakinkunmi/leaderboard.git
+- git clone git@github.com:ismailakinkunmi/leaderboard.git
 
-cd to the folder
+- cd to the folder
 
-cd Project-Webpack
+- cd Project-Leaderboard
+
```

Comment:
- [OPTIONAL] Please you have to install dependencies first by running `npm i` in the console, please kindly update that information in your `readme file`.

Reply:
@t-yanick thanks for the feedback.